mappings/net/minecraft/entity/mob/warden/WardenAi.mapping

```diff
@@ -0,0 +1,53 @@
+CLASS net/minecraft/unmapped/C_kazynfri net/minecraft/entity/mob/warden/WardenAi
+	FIELD f_bmrkldmz SCROLL_SPEED F
```

Comment:
Typo of stroll??

Reply:
Nope, that was me trying to work out the meaning of the field and forgetting to double check it 😄